src/services/fetchUsers.js

```diff
@@ -0,0 +1,26 @@
+import axios from 'axios';
+
+const getTokenFromLocalStorage = () => {
+  const storedData = localStorage.getItem('persist:auth');
+  if (!storedData) return null;
+
+  const parsedData = JSON.parse(storedData);
+  const accessToken = parsedData.accessToken;
```

Comment:
combine the property accessToken and replace methode to improve redability of code 
 const accessToken = parsedData.accessToken.replace(/\"/g, '');
 return accessToken